packages/components/src/components/Button/Button.stories.tsx

```diff
@@ -1,28 +1,42 @@
 import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import map from "lodash/map";
 import React from "react";
-import { Button } from "./Button";
+import { Box } from "../../primitives/Box";
+import { Button, ButtonProps } from "./Button";
 
 export default {
   component: Button,
   title: "Components/Button",
 } as ComponentMeta<typeof Button>;
 
-const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
+const variants = ["primary", "secondary", "ghost"];
```

Comment:
Why not import the Button variants here?

Reply:
Because it is a type and can't be used as a value.